pkg/controller/druid/handler.go

```diff
@@ -169,12 +169,23 @@ func deployDruidCluster(sdk client.Client, m *v1alpha1.Druid) error {
 					return nil
 				}
 
+				if m.Spec.ForcedRollbackSts == true {
+					if nodeSpec.PodManagementPolicy == "OrderedReady" {
+						checkCrashStatus(sdk, m)
+					}
+				}
+
 				// Check StatefulSet rolling update status, if in-progress then stop here
 				done, err := isStsFullyDeployed(sdk, nodeSpecUniqueStr, m)
 				if !done {
 					return err
 				}
 			}
+			if m.Spec.ForcedRollbackSts == true {
+				if nodeSpec.PodManagementPolicy == "OrderedReady" {
```

Comment:
minor nit: this can be translated to if A && B condition

Reply:
sure